LATEST

```diff
@@ -1,4 +1,5 @@
 ba6ba9019f0b3496fad68d936989fd9895cf3440 1.11.0-snapshot.20210217.6338.0.ba6ba901
+cc880e290b2311d0bf05d58c7d75c50784c0131c 1.10.1
```

Comment:
that doesn’t look right, this points to https://github.com/digital-asset/daml/commit/cc880e290b2311d0bf05d58c7d75c50784c0131c

Reply:
corrected